raddb/mods-available/client

```diff
@@ -37,15 +37,15 @@
 #
 #  The client files can be used as simple databases.  They can map from
 #  a client configuration item to an attribute at run-time via a map.
-#  This map can be placed in any `unlang` processing section/
+#  This map can be placed in any `unlang` processing section.
 #
 #  e.g:
 #
 #		map client <ipaddr> {
 #			Tmp-String-0 := 'nas_type'
 #			Tmp-String-1 := 'shortname'
 #			Tmp-String-3 += 'groups'	# Creates multiple attributes from custom group
-#							# attribute associated with the client.
+#										# attribute associated with the client.
```

Comment:
Re-indenting comments is not necessary.  I think your editor causes this.

Can you fix the whitespace issues and do a force push to update the commit?  I can then merge it.

Reply:
Ah! sorry for that. @alandekok I updated it.